src/epic.rs

```diff
@@ -0,0 +1,61 @@
+/// An Epic object contains all the information about an epic poem that we care
+/// about when we're generating messages.
+#[derive(Debug)]
+pub struct Epic {
+    /// The title of the epic poem
+    pub title: &'static str,
+    /// We store the Wikipedia article that we want to use for attribution. If a
+    /// given poem doesn't have an appropriate Wikipedia article to link to,
+    /// this will contain `None`.
+    pub wiki: Option<&'static str>,
+}
+
+/// A list of epic poems
+pub static EPICS: &[Epic] = &[
+    // 8th to 6th century BC
+    Epic {
+        title: "Epic of Gilgamesh",
+        wiki: Some("Epic of Gilgamesh"),
+    },
+    Epic {
+        title: "Atrahasis",
+        wiki: Some("Atra-Hasis"),
+    },
+    Epic {
+        title: "Enuma Elish ",
+        wiki: Some("Enuma Elish"),
```

Comment:
The Wikipedia title for this epic is "Enûma Eliš" and the link is https://en.wikipedia.org/wiki/En%C3%BBma_Eli%C5%A1, not "Enuma Elis"

Reply:
Huh interesting, the "List of" page has the one without the diacritics.

Reply:
It seems that the non-diacritical link (https://en.wikipedia.org/w/index.php?title=Enuma_Elish&redirect=no, can't link it directly) does redirect to the diacritical one, so you can use that one if you want to, I think.